week-03/README.md

```diff
@@ -17,68 +18,75 @@ The instructor will then review the pull request and make comments should furthe
 
 [Course Textbook Chapter 3, Problem 1]
 
-
+- *Question* (ldai4-stat6250): Why it is a good idea to begin DATA and PROC step in column?
+- *Answer* (ldai4-stat6250): Because a consistent layout enhances readability and enables you to understand the program’s purpose.
 
 [Course Textbook Chapter 3, Problem 2]
 
-
+- *Question* (ldai4-stat6250): If I didn’t find the spelling errors, what would happen after running the program?
 
 [Course Textbook Chapter 3, Problem 3]
 
+- *Question* (ldai4-stat6250): What is some common syntax errors?
+- *Answer* (ldai4-stat6250): Common syntax errors include spelling mistakes, omitting semicolons, leaving quotation marks unbalanced, and specifying invalid options.
 
- 
-[Course Textbook Chapter 3, Problem 4]
-
+ [Course Textbook Chapter 3, Problem 4]
```

Comment:
Existing content edited; please remove space added before problem header (in front of opening square brace) and DM me in Slack with the PR URL for re-evaluation